apps/api/src/controllers/topic.ts

```diff
@@ -20,9 +19,11 @@ class topicController {
     }
   }
   static async postsByTopicId(req: Request, res: Response, next: NextFunction) {
-    console.log(req.params["topicId"])
+    console.log(req.params["topicId"]);
     try {
-      const result = await topicService.postsByTopicId({topicId: req.params["topicId"]});
+      const result = await topicService.postsByTopicId({
+        topicId: req.params["topicId"],
```

Comment:
I'd prefer that we use dot syntax if possible (`req.params.topicId`) - If typescript is giving errors then this is fine.